src/compiler/scala/tools/nsc/ast/parser/Parsers.scala

```diff
@@ -380,11 +380,15 @@ self =>
           case DefDef(_, nme.main, Nil, List(_), _, _)  => true
           case _                                        => false
         }
+        def isApp(t: Tree) = t match {
+          case Template(ps, _, _) => ps.exists { case Ident(x) if x.decoded == "App" => true ; case _ => false }
```

Comment:
Thx, I grepped for it but missed it, trying to beat the evening rush home. Unless you mean add it to names.